pkg/apis/core/v1/helper/helpers_test.go

```diff
@@ -1509,3 +1509,98 @@ func TestHugePageUnitSizeFromByteSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLoadBalancerStatusEqual(t *testing.T) {
+
+	testCases := []struct {
+		left      *v1.LoadBalancerStatus
+		right     *v1.LoadBalancerStatus
+		name      string
+		expectVal bool
+	}{
+		{
+			name: "left equals right",
+			left: &v1.LoadBalancerStatus{
+				Ingress: []v1.LoadBalancerIngress{
+					{
```

Comment:
small nit: you can linewrap this into the previous and save one indent level :)

Reply:
The test casees in this file were all wrote in this style. I can format it to save one indent level.

Reply:
This is a big change. I think I can format it in a new PR if it's worth to do so.